src/extension/wps/src/main/java/au/org/emii/gogoduck/worker/GoGoDuckConfig.java

```diff
@@ -1,15 +1,77 @@
 package au.org.emii.gogoduck.worker;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Enumeration;
+import java.util.Properties;
+
 public class GoGoDuckConfig {
 
+    private static final Logger logger = LoggerFactory.getLogger(GoGoDuckConfig.class);
+    private static final String PROPERTIES_FILE = "config.properties";
```

Comment:
Should be consistent with other config used and use an xml config file preferrably with featuretype specific config in the featuretype directory